### ISSUE TITLE
dataset related question

### ISSUE THREAD

**Issue by haritsahm** [zylo117/Yet-Another-EfficientDet-Pytorch#575]
I want to use efficientdet for 1 class object detection. Is it a good idea to train and evaluate with both positive images (images with > 0 objects) and negative images (images with 0 objects) ?

**Comment by zylo117** [comment]
images with 0 object will not be trained

**Comment by haritsahm** [comment]
> images with 0 object will not be trained

But in my notebook, the total iteration for each epoch is the total images (positive+negative)/batch size

**Comment by zylo117** [comment]
Yes, but those images with no gt box will be skipped.
https://github.com/zylo117/Yet-Another-EfficientDet-Pytorch/blob/master/efficientdet/loss.py#L56

**Comment by haritsahm** [comment]
> 
> 
> Yes, but those images with no gt box will be skipped.
> https://github.com/zylo117/Yet-Another-EfficientDet-Pytorch/blob/master/efficientdet/loss.py#L56

I see, thank you!
